package.json

```diff
@@ -110,6 +113,7 @@
     "lint-staged": "^8.1.0",
     "node-persist": "^3.0.4",
     "nodemon": "^1.18.9",
+    "npm-run-all": "^4.1.5",
```

Comment:
Does one need npm-run-all and concurrently? They seem to cover pretty much the same functionality as far as I can tell.